lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -104,11 +104,142 @@ String FunctionalChaosResult::__repr__() const
+  const UnsignedInteger inputDimension = distribution_.getDimension();
+  oss << getClassName() << "\n"
+      << "- input dimension=" << inputDimension << "\n"
+      << "- output dimension=" << outputDimension << "\n"
+      << "- distribution dimension=" << distribution_.getDimension() << "\n"  
+      << "- transformation=" << transformation_.getInputDimension() << " -> " << transformation_.getOutputDimension() << "\n"
+      << "- inverse transformation=" << inverseTransformation_.getInputDimension() << " -> " << inverseTransformation_.getOutputDimension() << "\n"
+      << "- orthogonal basis dimension=" << orthogonalBasis_.getMeasure().getDimension() << "\n"
+      << "- indices size=" << indicesSize << "\n"
+      << "- relative errors=" << relativeErrors_ << "\n"
+      << "- residuals=" << residuals_ << "\n";
+  oss << "\n";
+
+  const Scalar ell_threshold = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintEllipsisThreshold");
+  const UnsignedInteger ell_size = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintEllipsisSize");
+  const UnsignedInteger columnWidth = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintColumnWidth");
+  const UnsignedInteger ellipsis = (indicesSize * outputDimension > ell_threshold);
+  const bool isEllipsisEnabled = (ellipsis and outputDimension > 2 * ell_size);
+  // Print table header
+  oss << "| Index |" << padString(" Multi-index ") << "|";
+  String intermediateString;
+  if (isEllipsisEnabled)
+  {
+    for (UnsignedInteger j = 0 ; j < ell_size; ++ j)
+    {
+      intermediateString = OSS() << " Coeff.#" << j;
+      oss << padString(intermediateString) << "|";
+    }
+    oss << padString(" ... ") << "|";
+    for (UnsignedInteger j = outputDimension - ell_size ; j < outputDimension; ++ j)
+    {
+      intermediateString = OSS() << " Coeff.#" << j;
+      oss << padString(intermediateString) << "|";
+    }
+    oss << "\n";
+  }
+  else
+  {
+      if (outputDimension == 1) 
+        oss << padString(" Coefficient ") << "|\n";
+      else
+      {
+        for (UnsignedInteger j = 0 ; j < outputDimension; ++ j)
+        {
+          intermediateString = OSS() << " Coeff.#" << j;
+          oss << padString(intermediateString) << "|";
+        }
+        oss << "\n";
+      }
+  }
+  // Print dashes
+  String dashesSeparator = String(columnWidth, '-') + "|";
+  oss << "|-------|" << dashesSeparator;
+  if (isEllipsisEnabled) oss << repeatString(1 + 2 * ell_size, dashesSeparator) << "\n";
+  else if (outputDimension == 1) oss << dashesSeparator << "\n";
+  else oss << repeatString(outputDimension, dashesSeparator) << "\n";
+  // Print table content: multi-indices
+  for (UnsignedInteger i = 0; i < indicesSize; ++ i)
+  {
+    if (ellipsis && indicesSize > 2 * ell_size)
+    {
+      if (i == ell_size)
+      {
+        oss << "| ...   |";
+        const String blankColumn = String(columnWidth, ' ') + "|";
```

Comment:
String is a complex type. Use `const String blanckColumn(String(columnWidth, ' ') + "|");`